specs/accounting_reform/beacon-chain.md

```diff
@@ -224,19 +232,18 @@ def get_standard_flag_deltas(state: BeaconState,
     total_participating_balance = get_total_balance(state, unslashed_participant_indices) // increment
     total_balance = get_total_active_balance(state) // increment
     for index in get_eligible_validator_indices(state):
+        base_reward = get_base_reward(state, index)
         if index in unslashed_participant_indices:
             if is_in_inactivity_leak(state):
                 # Since full base reward will be canceled out by inactivity penalty deltas,
                 # optimal participation receives full base reward compensation here.
-                rewards[index] += get_base_reward(state, index) * numerator * total_balance // REWARD_DENOMINATOR
+                rewards[index] += base_reward * numerator // REWARD_DENOMINATOR
             else:
                 rewards[index] += (
-                    (get_base_reward(state, index) * total_participating_balance // total_balance)
+                    (base_reward * total_participating_balance // total_balance + base_reward)
```

Comment:
What's the extra `+ base_reward` doing here?

Reply:
Oh I see, basically everyone always gets penalized for one base reward, and this is to compensate for that?